contracts/token/ERC721/ERC721.sol

```diff
@@ -25,17 +25,13 @@ abstract contract ERC721 is Context, ERC165, IERC721, IERC721Metadata, IERC721Er
     // Token symbol
     string private _symbol;
 
-    // Mapping from token ID to owner address
-    mapping(uint256 => address) private _owners;
+    mapping(uint256 tokenId => address owner) private _owners;
 
-    // Mapping owner address to token count
-    mapping(address => uint256) private _balances;
+    mapping(address owner => uint256 tokenCount) private _balances;
```

Comment:
why not simply balance ?

Reply:
I just didn't want to be redundant, I changed it to balance.